classes/class-uabb-compatibility.php

```diff
@@ -6,7 +6,11 @@
  * @package BB Version Check
  */
 
-if ( ! class_exists( 'UABB_Lite_Compatibility' ) ) {
+if ( ! class_exists( 'FL_BUILDER_VERSION' ) ) {
```

Comment:
Here we need to check UABB_Lite_Compatibility class only

Reply:
reverted changes